### PR TITLE
Update

### PR DESCRIPTION
**Pull request by Justman100**

### CODE REVIEW

File: docker-compose-https.yml [hunk 1]
```diff
@@ -2,6 +2,9 @@ version: '3'
 services:
   weblate:
     image: weblate/weblate
+    # To use a custom port, change the first 8080 to your port
+    ports:
+    - 8080:8080
```
Comment (nijel) [inline docker-compose-https.yml:7]: There is no need to expose port here, it is done via `https-portal` below.

File: docker-compose-https.yml [hunk 2]
```diff
@@ -19,14 +22,14 @@ services:
       WEBLATE_ENABLE_HTTPS: 1
       WEBLATE_IP_PROXY_HEADER: HTTP_X_FORWARDED_FOR
   database:
-    image: postgres:16-alpine
+    image: postgres:16.2
```
Comment (nijel) [inline docker-compose-https.yml:25]: Why do you think pinning major version is not good enough here? We're not really playing on the exact environment here (Weblate itself is not pinned at all).

Also, I don't see a reason for stopping using alpine variant, which is smaller.

**Review by nijel — COMMENTED** [review]
I'd really prefer to keep the scripts as simple as possible, so I'd drop support for `docker-compose` and support `docker compose` only. I've just done that in https://github.com/WeblateOrg/docker-compose/pull/232. The yaml files changes are mostly not desired.

File: docker-compose-https.yml (hunk 1 above)
Comment (Justman100) [reply docker-compose-https.yml:7]: > There is no need to expose port here, it is done via `https-portal` below.

Yes, I could not access Weblate until I published the ports

File: docker-compose-https.yml (hunk 2 above)
Comment (Justman100) [reply docker-compose-https.yml:25]: > Why do you think pinning major version is not good enough here?

Because 16.2 is newer than 16!

Comment (nijel) [reply docker-compose-https.yml:25]: No, 16 is a moving tag for the latest release in 16.x series, see https://hub.docker.com/_/postgres.

And I also ask for removal of the `-alpine` suffix.

Comment (Justman100) [reply docker-compose-https.yml:25]: > No, 16 is a moving tag for the latest release in 16.x series, see https://hub.docker.com/_/postgres.

So, the _16_ tag automatically takes the latest sub-version of the 16.x series?

**Pull request closed without merging**